feedhenry/fh-fhc
# FH-3874 - Tests into the root of fh3 directory have not been called into the unit tests

JIRA: https://issues.jboss.org/browse/FH-3874

**Changes**
- Fix path into Gruntfile.js in order to run/call all test files into the directory unit/test/fh3
- Fix all tests which were into the /
- Fix bug related to the import of zip files. ( This bug was not found before because of this its unit have been not called )

**Step to verify**
- Check into the build of the Jenkins that the files into the root of test/unit/fh3/ dir were called - E.g  - Looking for test_ping.js it into the print of console.log
